### ISSUE TITLE
How to..

### ISSUE THREAD

**Issue by blondie63** [madhukesh048/Flutter-MVVM-VS-Code-extension-Pack#5]
Please add a little how to use this extension.. :)

**Comment by blondie63** [comment]
Sorry but i don't understand.. where is this documentation ?
On readME.md i don't see any commands..
Looking on medium i see something like "Flutter Architecture: Create Widget"
But is this a command ?
I've tried but..
Could not find a command named "Architecture:".

**Comment by ajilo297** [comment]
Once the extension is installed you can press `Ctrl` (`Cmd` on Mac) + `Shift` + `P` to open the command palette and type in any of the following commands. It should show up as autocomplete.

* Flutter Architecture: Initialize Architecture
* Flutter Architecture: Create Views
* Flutter Architecture: Create Widgets

In hindsight, I can see why it might be hard to follow along, just based on the `README.md`. I'll update it over the weekend with more detailed steps and caveats.

**Comment by blondie63** [comment]
ok, but there's a text bug on extension.. look:
<img width="548" alt="flutter-ext-mvvm-bug" src="https://user-images.githubusercontent.com/1261851/72201412-01c7f400-3454-11ea-9f6b-d0176accf06c.png">

**Comment by blondie63** [comment]
I also recommend adding the fact that you must first create an empty flutter project and then run the commands of your extension
